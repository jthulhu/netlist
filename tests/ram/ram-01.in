00
0
00
0000
01
0
00
0000
10
0
00
0000
11
0
00
0000
00
1
11
1100
11
1
01
0011
01
1
01
0100
01
0
10
1111
10
0
01
1000
01
0
00
0000
