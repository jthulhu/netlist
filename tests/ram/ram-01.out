=> o = <0000>
=> o = <0000>
=> o = <0000>
=> o = <0000>
=> o = <0000>
=> o = <1100>
=> o = <1111>
=> o = <1111>
=> o = <0000>
=> o = <1111>
